openpype/hosts/maya/plugins/load/_load_animation.py

```diff
@@ -7,7 +7,7 @@ class AbcLoader(openpype.hosts.maya.api.plugin.ReferenceLoader):
     families = ["animation",
                 "camera",
                 "pointcache"]
-    representations = ["abc"]
+    representations = ["abc", "fbx"]
```

Comment:
This is a bit weird - the Loader class is called `AbcLoader`, it forces a load of `AbcImport.mll` but for FBX should load `fbx4maya.mll` or whatever the plugin name is. I feel like we're better off with a dedicated FBX loader, no?

Without the FBX plugin loaded this loader will likely fail to load the FBX?